### ISSUE TITLE
Multiple discard reader tags in a row broken

### ISSUE THREAD

**Issue by madstap** [kkinnear/zprint#58]
When using `#_#_` to comment out  multiple forms in a row, one of the forms disappear.

Repro:

```clojure
(zp/zprint-file-str "{:foo :bar #_#_:baz :quux}" nil {}) ;=> "{:foo :bar, #_#_:baz}"


(zp/zprint-file-str "[1 #_#_#_2 3 4]" nil {}) ;=> "[1 #_#_#_2]"

```

**Comment by kkinnear** [comment]
Wow, it was only about two weeks ago that I even found out that was legit Clojure syntax! 

I can reproduce the problem, and while at first I thought it might be something in the parser library that I'm using, so far it looks like it is doing something reasonable with the multiple uneval tokens.  

I'm thinking that I will be able to work with what it is giving me and do something reasonable with the multiple uneval tokens.  But it isn't going to be a walk in the park, so you might want to hold off on using this syntax with zprint for a bit.  I'm going to work on this next, but it will be next week at the earliest before I have much time to spend on it.  

Thanks for noticing this issue!

**Comment by kkinnear** [comment]
Fixed in 0.4.10.
